support-frontend/typescript-errors.json

```diff
@@ -732,6 +731,10 @@
             "path": "assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx",
             "message": "Parameter 'result' implicitly has an 'any' type."
         },
+        {
+            "path": "assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx",
+            "message": "Parameter 'result' implicitly has an 'any' type."
+        },
```

Comment:
I think maybe running `yarn` to install latest deps will fix this one

Reply:
I think you'll just need to run yarn in the `support-frontend` sub dir & then `yarn ts-errors` to regenerate this file

Reply:
Done :)